pkgs/tools/misc/go-ios/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "go-ios";
+  version = "1.0.114";
+
+  src = fetchFromGitHub {
+    owner = "danielpaulus";
+    repo = "go-ios";
+    rev = "v${version}";
+    sha256 = "sha256-tOaoZkbLObJJb2oTeCE1JowZ0QySKlpHXCiplGVlCMw=";
+  };
+
+  vendorSha256 = "sha256-GtXnjK3tAEIj4l0HeXYjdJ4MWq824euKkfiy/vA76Sw=";
+
+  postPatch = ''
+    # do not build this (does not work)
+    rm -rf restapi
+
+    # remove these tests, as they fail
+    find . -name '*_integration_test.go' -delete
+    rm ios/testmanagerd/xcuitestrunner_test.go
+  '';
```

```suggestion
  excludedPackages = [
    "restapi"
  ];

  checkFlags = [
    "-tags=fast"
  ];
```

Upstream flags those integration/failing tests with build tags, which we can re-use :)